circuit_definitions/README.md

```diff
@@ -0,0 +1,55 @@
+# Circuits definitions crate
+
+This crate contains the 'outer layer' for multiple circuits. The concrete circuits code is in `era-zkevm_circuits`
+repository.
+
+## Code structure
+
+We have 13 different 'base layer' circuits (for example MainVM, Decomitter), and 3 recursive circuits (Leaf, Node and
+Scheduler).
+
+Base layer circuits are located in `src/base_layer`, Recursive circuits are in `src/recursion_layer`.
+
+We also have 2 'AUX' circuits: compressor and wrapper, that are run on top of the final Scheduler proof, and they are
```

Comment:
I would say that we have a couple of compression circuits and one wrapper in the end. The final compressor should be proved with Bn256-Poseidon2 hash. For now, we use only one compressor. (it could be hard for readers to understand, why do we need 10 types of compression circuits: 5 with Goldilocks hash and 5 with Bn256 hash)